examples/linux64/wflow_piave_subbasin/log.txt

```diff
@@ -0,0 +1,106 @@
+[ Info: Wflow version `v0.7.2`
+[ Info: Initialize model variables for model type `sbm`.
+[ Info: Cyclic parameters are provided by `staticmaps.nc`.
+[ Info: Forcing parameters are provided by `inmaps.nc`.
+[ Info: Set `vertical.precipitation` using NetCDF variable `precip` as forcing parameter.
+[ Info: Set `vertical.temperature` using NetCDF variable `temp` as forcing parameter.
+[ Info: Set `vertical.potential_evaporation` using NetCDF variable `pet` as forcing parameter.
```

Comment:
Should this log file be included?

Reply:
A case can be made for both I suppose, but given it's testdata that has been produced with HydroMT, I'd say: leave it in there.